From 84c772c1d03d57ca5760e68c817648dc793c0f1b Mon Sep 17 00:00:00 2001
From: GammaC0de <gammac0de@users.noreply.github.com>
Date: Sun, 27 Jun 2021 22:37:46 +0300
Subject: [PATCH 120/150] log exceptions on captcha tasks

---
 src/pyload/core/managers/captcha_manager.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

--- a/src/pyload/core/managers/captcha_manager.py
+++ b/src/pyload/core/managers/captcha_manager.py
@@ -51,7 +51,11 @@ class CaptchaManager:
             try:
                 plugin.captcha_task(task)
             except Exception:
-                pass
+                self.pyload.log.warning(
+                    self.pyload._("Unable to create captcha task"),
+                    exc_info=self.pyload.debug > 1,
+                    stack_info=self.pyload.debug > 2
+                )
 
         if task.handler or cli:  #: The captcha was handled
             self.tasks.append(task)
